components/Projects/Project.tsx

```diff
@@ -12,35 +12,38 @@ export const Project = ({ title, description, tags, author }: ProjectProps) => {
   const handleContribute = () => {
     return;
   };
+  const [innerWidth, setInnerWidth] = useState(0);
+  useEffect(() => {
+    setInnerWidth(window.innerWidth);
+  }, []);
   return (
     <div className="w-full p-2 md:p-0">
       <div
         className={`${
           mode
             ? 'text-white bg-box-color border border-[#363D45]'
             : 'text-dark-color border border-[#C9C9C9]'
-        } text-white shadow-md rounded-md items-center md:flex justify-center space-y-2`}
+        } text-white shadow-md rounded-md items-center flex-col md:flex md:flex-row justify-center space-y-2`}
       >
-        <div className="w-[250px] md:w-[500px] mx-auto h-auto p-4">
+        <div className="w-full p-2 flex justify-center items-center">
           <Image
             src={'/open-source.png'}
             alt={title}
-            height={'300'}
-            width={'300'}
-            className="object-contain  "
+            height={800}
+            width={innerWidth/1.2}
```

Comment:
you should put object-contain as the image user will provide can be of  different dimensions

Reply:
object-cover should be best here it will cover the entire area.

Reply:
yeah but what if user provide image which is not of the same dimension in that case it can look bad .

Reply:
if user uploads different dimension it will cover the entire image so it will fit to the container